### PR TITLE
Fix #53 Upgrade to be able to set token expiration

### LINKED ISSUE #53: cli-create-embed-token cannot set expiration
Hello,

I want to set a token expiration date by cli-create-embed-token.
`createReportEmbedToken` function of powerbi-api can set expiration (see below), but cli-create-embed-token doesn't set any expiration parameter to it.

**https://github.com/Microsoft/PowerBI-Node/blob/master/lib/core/powerBIToken.ts#L17**

### PR DESCRIPTION
**Pull request by kmorihiro**
Add a feature of setting expiration to cli-create-embed-token.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,15 +84,15 @@ Updates connection strings and/or credentials for an existing dataset
...
 
-`powerbi create-embed-token -c <collection> -k <accessKey> -w <workspaceId> -d <datasetId> -u [username] --roles [roles1,roles2,...] -s [scope1 scope2 ...]`
+`powerbi create-embed-token -c <collection> -k <accessKey> -w <workspaceId> -d <datasetId> -u [username] --roles [roles1,roles2,...] -s [scope1 scope2 ...] -e <expiration>`
 
 > Examples
 
 To create a Power BI embed token with specific user and one role:
 
-`powerbi create-embed-token -c <collection> -k <key>...key -w 06640...744d8f10  -r 07842...e30  -u "Ali Hamud" --roles "Developer"`
+`powerbi create-embed-token -c <collection> -k <key>...key -w 06640...744d8f10  -r 07842...e30  -u "Ali Hamud" --roles "Developer" -e "2020-01-01"`
```
Comment (ali-hamud) [inline README.md]: please give example of a full time here:
03-02-2018 15:03:54 UTC

File: lib/cli-create-embed-token.ts [hunk 2]
```diff
@@ -51,7 +52,8 @@ export default function CliCreateEmbedToken() {
             if (!(settings.report || settings.dataset)) {
                 return cli.error('Either reportId or datasetId must be set');
             }
-            token = powerbi.PowerBIToken.createReportEmbedToken(settings.collection, settings.workspace, settings.report, settings.dataset, settings.scopes, settings.username, settings.roles);
+            let expiration = settings.expiration ? new Date(settings.expiration) : void 0;
+            token = powerbi.PowerBIToken.createReportEmbedToken(settings.collection, settings.workspace, settings.report, settings.dataset, settings.scopes, settings.username, settings.roles, expiration);
```
Comment (ali-hamud) [inline lib/cli-create-embed-token.ts]: why void 0?
please sett null

File: README.md (hunk 1 above)
Comment (kmorihiro) [inline README.md]: Thank you for your review and sorry for the delay. I fixed an example to a full-time.

File: lib/cli-create-embed-token.ts (hunk 2 above)
Comment (kmorihiro) [inline lib/cli-create-embed-token.ts]: `expiration` parameter of `createReportEmbedToken` should be Date object or no value (undefined, void 0). In fact, `expiration` is initialized to null by 'createReportEmbedToken' if it's undefined, but it's better to observe function parameter type, IMHO.

**Pull request merged**